src/StdStyle.sol

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
+
+import {Vm} from "./Vm.sol";
+
+library StdStyle {
+    Vm private constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    string constant RED = "\u001b[91m";
+    string constant GREEN = "\u001b[92m";
+    string constant YELLOW = "\u001b[93m";
+    string constant BLUE = "\u001b[94m";
+    string constant MAGENTA = "\u001b[95m";
+    string constant CYAN = "\u001b[96m";
+    string constant BOLD = "\u001b[1m";
+    string constant DIM = "\u001b[2m";
+    string constant ITALIC = "\u001b[3m";
+    string constant UNDERLINE = "\u001b[4m";
+    string constant REVERSE = "\u001b[7m";
```

Comment:
Would `INVERT` or `INVERSE` be a better name for this? IIUC it inverts the colors

Reply:
I thought of this, but I decided to follow the standard naming for ANSI/VT100:
- https://misc.flogisoft.com/bash/tip_colors_and_formatting
- https://www2.ccs.neu.edu/research/gpc/VonaUtils/vona/terminal/vtansi.htm

However I agree that `invert`or `inverse` could be better options as `reverse` may be confused with the text being reversed.